### ISSUE TITLE
Fence in data import with integrity checks

### ISSUE THREAD

**Issue by alexsoble** [studentinsights/studentinsights#147]
Via slack conversation with @kevinrobinson:

* Write queries to fence in what we expect the database to look like
* Run as a task after data import
* If it detects something is wrong in our data universe, it calls us

**Comment by alexsoble** [comment]
Ran against production data.

```
Uh oh! Data check failed:
Validation failed: Student can't be blank
```

Will investigate! And keep running until all the data is valid. :blush:

**Comment by alexsoble** [comment]
All data integrity checks now pass. Best of all, they helped identify and fix another data problem (#185).

Good suggestion @kevinrobinson!
